pysiaf/aperture.py

```diff
@@ -1437,8 +1437,17 @@ def verify(self):
                 print('Verification WARNING: {} has non-integer sum between YDetRef {} and '
                       'YSciRef {} DetSciYAngle {}'.format(self.AperName, self.YDetRef, self.YSciRef,
                                                           self.DetSciYAngle))
-
-
+    def dms_corner(self):
+        """Compute the pixel value of the lower left corner of an aperture.
+        
+        This corresponds to the OSS corner position (x: ColCorner, y: RowCorner).
+        The notation for OSS is 1-based, i.e. the lower left corner of a FULL subarray is (1,1)
+        """
+        ColCorner = np.ceil(np.min(self.corners('det')[0])).astype(np.int_)
```

Comment:
Adhere to PEP8 variable naming conventions: use `column_corner` and `row_corner` names.

Reply:
done, and spotted and fixed the same in enhanced_aperture_file. Also fixed an other couple of instances (those I could spot) of non pep8 compliant variable names